templates/CRM/Core/BillingBlock.tpl

```diff
@@ -46,7 +46,7 @@
       <label for="billingcheckbox">{ts}My billing address is the same as above{/ts}</label>
     {/if}
     <fieldset class="billing_name_address-group">
-      <legend>{ts}Billing Name and Address{/ts}</legend>
+      <legend>{ts}Your Registration Info{/ts}</legend>
```

Comment:
@pradpnayak was this added to the commit by mistake?

Reply:
sorry, i have updated the PR.